Swap Indexes API (#192)
meilisearch/specifications: text/0061-error-format-and-definitions.md

## index_not_found

> `Asynchronous` / `Synchronous`

Author: This error can be synchronous @Kerollmops. e.g `GET /indexes/thisonedoesnotexists`

---

## duplicate_index_found

`Asynchronous`

### Context

> This error happens when the same indexUid is found twice for a `POST`- `swap-indexes` payload.

Reviewer: Not really clear, I thought the error was if two indexes have the same uid

Reviewer: ```suggestion
This error happens when the same indexUid is used twice in the `POST`- `swap-indexes` payload.
```
